server/server_test.go

```diff
@@ -0,0 +1,84 @@
+// Package server
+package server
+
+import (
+	"go.uber.org/zap"
+
+	"github.com/kardiachain/kardia-explorer-backend/metrics"
+	"github.com/kardiachain/kardia-explorer-backend/types"
+)
+
+func createDevSrv() (*Server, error) {
+	cfg := Config{
+		StorageAdapter:     "mgo",
+		StorageURI:         "mongodb://kardia.ddns.net:27017",
+		StorageDB:          "explorerTest",
+		StorageIsFlush:     true,
+		MinConn:            4,
+		MaxConn:            16,
+		KardiaURLs:         []string{"https://dev-1.kardiachain.io"},
+		KardiaTrustedNodes: []string{"https://dev-1.kardiachain.io"},
+		CacheAdapter:       "redis",
+		CacheURL:           "kardia.ddns.net:6379",
+		CacheDB:            0,
+		CacheIsFlush:       true,
+		BlockBuffer:        20,
+		HttpRequestSecret:  "httpTestSecret",
+		VerifyBlockParam: &types.VerifyBlockParam{
+			VerifyTxCount:   false,
+			VerifyBlockHash: false,
+		},
+		Metrics: metrics.New(),
+		Logger:  zap.L(),
+	}
+
+	return New(cfg)
+}
+
+func createTestSrv() (*Server, error) {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`createTestSrv` is unused (deadcode)